src/Servant/OAuth/JWT.hs

```diff
@@ -118,18 +107,45 @@ data JWTSignSettings = JWTSignSettings
     jwtInitialClaims :: ClaimsSet,
     jwtDuration :: NominalDiffTime
   }
+  deriving (Eq, Show, Generic)
+
+instance FromJSON JWTSignSettings
+
+instance ToJSON JWTSignSettings
+
+-- | Generate a simple set of crypto credentials for a token server.  Get familiar with `jose`
+-- if you want a `JWTSignSettings` that is ready for production.  Start with
+-- `hs-jose/example/Main.hs`.
+mkTestJWTSignSettings :: MonadIO m => m JWTSignSettings
```

Comment:
This is only used in test code: Shouldn't it be in some module in the `test` folder? :thinking: